examples/config_dc2_steps.yaml

```diff
@@ -0,0 +1,121 @@
+# This is an example of the template specification file for Steps used in DC2 campaigns
+
+# Import the common element templates
+- import: "${CM_CONFIGS}/config_standard_elements.yaml"
+# Now define the steps
+- SpecBlock:
+      name: dc2_step1
+      includes: ['step']
+      data:
+          pipeline_yaml: "${DRP_PIPE_DIR}/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step1"
+      child_config:
+          spec_block: group
```

Comment:
What is the difference between `SpecBlock` and `spec_block` ? I'm concerned about confusion in editing these configs.

Reply:
SpecBlock defines a thing, spec_block refers that thing by name.

Reply:
Would it perhaps be a little more clear to call spec_block something like parent_spec_block? Or am I misinterpreting the use of it?

Reply:
`parent` is a little overloaded here in that it also refers to the parent node in the hierarchy.    Here and elsewhere `spec_block` is refering to "the SpecBlock used to build that child".